### ISSUE TITLE
Anpassung Ornamentation

### ISSUE THREAD

**Issue by alingnau** [saw-leipzig/kucha#190]
Ziel: Suche der Ornamentik ähnlich wie in den anderen Sektoren, insbesondere wie bei "Pictorial Elements"):

einfache Suche:
- Ornament Code
- Cave
- Ornamental Motif

erweiterte Suche:
- Districts
- Cave Part (damit ist entweder Wände oder Decke gemeint)
- Position
- Function (nur wenn man die Position ausgewählt hat)
- Ornament Component

Noch zwei Felder würde ich lieber erst unsichtbar lassen (da sie die Suche unnötig kompliziert machen wurden), und zwar:
- Inner Secondary Component
- Related Ornament

**Comment by LucaDeF** [comment]
- [ ] Suchfunktion für "Sites"

**Comment by LucaDeF** [comment]
- [ ] Die Ergebnisse der Suchfunktion sollten nach dem Code des Ornaments sortiert werden (ich denke, es ist jetzt nach ID).

**Comment by erikradisch** [comment]
Das kann geschlossen werden, oder?

**Comment by LucaDeF** [comment]
Soviel ich sehe, funktioniert die Suchfunktion eigentlich noch nicht. Es findet was, nur wenn man nach die Motive oder die Iconographie sucht!
Ausserdem, die ganze Funktion muss neu organisiert werden: mit der neuen Organisation sind einige Felder sinnlos. 
Ich würde die Filterfelder begrenzen. Nämlich würde ich die nutzen, die auch für die Painted Represantation laufen, und zwar:
- [ ] Orament code (Äquivalent von "short name")
- [ ] Cave search
- [ ] Location search
- [ ] Iconography
- [ ] Wall search
- [ ] Position

ich denke, es könnte für die Ornamente auch nutzlich sein, wenn man nach die "components" suchen könnte.

Ich weiß es aber jetzt nicht, ob es sich lohnt, die Suchfunktion zu bearbeiten, solange den Editor nicht komplet geschlossen ist!! (Suchfelder hängen schliesslich von den Editorfelder ab, oder?)

**Comment by LucaDeF** [comment]
noch zwei Suchfelder hinzufügen:
- [ ] "components"
- [ ] Suchfunktion mit Text für das Feld "Position note"

**Comment by LucaDeF** [comment]
Soviel ich sehe folgenden Punkt ist eigentlich noch nicht erledigt:
- [ ] Die Ergebnisse der Suchfunktion sollten nach dem Code des Ornaments sortiert werden (ich denke, es ist jetzt nach ID).

**Comment by erikradisch** [comment]
was meinst Du denn mit location search?

**Comment by erikradisch** [comment]
braucht es das mit den sites wirklich?

**Comment by LucaDeF** [comment]
> was meinst Du denn mit location search?

Quatsch...das war wohl einen Fehler von mir: in der neue Organisation brauchen wir doch keinen "Location"-Filter in der Ornamentik Abteilung!!

**Comment by LucaDeF** [comment]
> braucht es das mit den sites wirklich?

Ich finde es nützlich, aber wenn es nicht für die "Painted Representation" vorgesehen ist, hat es wahrscheinlich auch keinen Sinn nur für die Ornamentik Abteilung es zu bauen!!

**Comment by erikradisch** [comment]
Einzig fehlt ja die Position Note hier, oder? Diese sollte dann wohl besser in die painted representation eingebunden werden, oder?

**Comment by LucaDeF** [comment]
Sowie die Abteilungen jetzt organisiert sind, ist es wahrscheinlich besser! In diesem Fall muss es aber mit den anderen besprechen werden.

**Comment by LucaDeF** [comment]
Jetzt dass diese Abteilung als "Ideal Typical Pictures" gilt, habe ich mich wieder die Filter Felder angeguckt.
Drei Felder sind jetzt vollkommen zwecklos und ICH würde dann sie ausblenden:

- [ ] Motif (dieser Feld existiert nicht mehr in der Karteikarte: die Baumstruktur hat den ersetzt)
- [ ] Text Search (dieser Feld war nie nutzlich)
- [ ] Components of Iconography

**Comment by erikradisch** [comment]
So. Hier habe ich auch mal aufgeräumt. Components of Iconography und Motif sind gelöscht. Text-Search habe ich gelassen, da die eigentlich funtkioniert. Wobei die Textsuche im Frontend natürlich viel cooler ist. ;-)
